nss-day-cohort-60/rare-client-tenacious-tators
# Jes search posts

Closes #2


# Description of Proposed Changes

Search bar added to PostList.js

---


# Steps to Test

1. Fetch the jes-search-posts branch on both server AND client and switch to them, start and run debugger
2. navigate to /posts
3. enter a search term to search by title, any portion of a word should return the search. Test by clicking button, and hitting enter
4. Test that View All button returns all the posts to view
